scheds/rust/scx_rusty/src/bpf/main.bpf.c

```diff
@@ -1262,6 +1261,12 @@ static u64 node_dom_mask(u32 node_id)
 static void task_set_preferred_mempolicy_dom_mask(struct task_struct *p,
 						  struct task_ctx *taskc)
 {
+#warning "FIXME - See the comment"
+	/*
+	 * XXX - p->mempolicy no longer exists on v6.13. Disable mempolicy
```

Comment:
I guess this could use `if (bpf_core_field_exists(p->mempolicy))` , but most likely not being used. Maybe we can make a help wanted task to fix it.